### PR TITLE
Refactor Events and Venue MVC, closes #108

### PR DESCRIPTION
**Pull request by KristaANelson**
Primary change was to make and event table partial which was used for the events index and venue show view. I also made several event methods in order to try to take some logic out of the view and controller and move it to the model. 

I also made the available tickets links  into buttons so it would be easier to see and entice the view to click on it. 

all tests are passing 
screen shot 2015-02-19 at 2 53 50 am
screen shot 2015-02-19 at 2 54 07 am
screen shot 2015-02-19 at 2 54 24 am

### CODE REVIEW

File: app/controllers/venues_controller.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@ class VenuesController < ApplicationController
 
   def show
     @venue = Venue.find_by(id: params[:id])
-    @events = Event.active_events(@venue)
+    @events = @venue.open_events
```
Comment (scottcrawford03) [inline app/controllers/venues_controller.rb]: I don't know that this works. We're calling an event method on the a venue object. The purpose for the original code on the left was to specify active events for a specific venue.

File: app/models/event.rb [hunk 2]
```diff
@@ -17,8 +17,35 @@ class Event < ActiveRecord::Base
   scope :open_events, -> { where("date >= ?", Date.today).is_approved }
   scope :is_approved, -> { where approved: true }
 
-  def self.active_events(venue)
-    active.where("venue_id = ?", venue.id)
```
Comment (scottcrawford03) [inline app/models/event.rb]: again, I think we may actually need this method.

File: app/controllers/venues_controller.rb (hunk 1 above)
Comment (scottcrawford03) [inline app/controllers/venues_controller.rb]: just saw that you pulled this into venue.rb as well. I like it.

File: app/models/venue.rb [hunk 3]
```diff
@@ -4,4 +4,8 @@ class Venue < ActiveRecord::Base
   validates :location, presence: true
 
   has_many :events
+
+  def open_events
+    events.open_events
```
Comment (scottcrawford03) [inline app/models/venue.rb]: this is nice.

**Comment by bhargavisatpathy** [comment]
Nice refactor and pulling out partials :+1:

**Pull request merged**